azext_iot/_help.py

```diff
@@ -288,21 +302,37 @@
     long-summary: For more information, see
                   https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
     examples:
+    - name: Import all device identities from a blob by providing command parameters for
```

Comment:
Should we state what the input blob name is that is expected to be present in the container?